browser/src/Editor/SimpleNeovimEditor.tsx

```diff
@@ -0,0 +1,120 @@
+/**
+ * NeovimEditor.ts
+ *
+ * IEditor implementation for Neovim
+ */
+
+import * as React from "react"
+
+import { IncrementalDeltaRegionTracker } from "./../DeltaRegionTracker"
+import { NeovimInstance } from "./../neovim"
+import { CanvasRenderer, DOMRenderer, INeovimRenderer } from "./../Renderer"
+import { NeovimScreen } from "./../Screen"
+
+import * as Config from "./../Config"
+
+// import { PluginManager } from "./../Plugins/PluginManager"
+
+import { Keyboard } from "./../Input/Keyboard"
+import { IEditor } from "./Editor"
+
+import { NeovimRenderer } from "./NeovimRenderer"
+
+export class DummyPluginManager {
+    public getAllRuntimePaths(): string[] {
+        return []
+    }
+
+    public startPlugins(neovimInstance: NeovimInstance): void {
+        // no-op
+    }
+}
+
+export class SimpleNeovimEditor implements IEditor {
+
+    private _neovimInstance: NeovimInstance
+    private _deltaRegionManager: IncrementalDeltaRegionTracker
+    private _renderer: INeovimRenderer
+    private _screen: NeovimScreen
+
+    private _pendingTimeout: any = null
+    private _pendingAnimationFrame: boolean = false
+
+    constructor(
+        private _config: Config.Config = Config.instance(),
+    ) {
+        // TODO: How to get rid of this?
+        this._neovimInstance = new NeovimInstance(new DummyPluginManager(), 100, 100)
```

Comment:
Yeah I agree, there a few ways to deal with it... I wonder. Do we want separate neovim instances per multiplex? We could have just one and manage it that way, this might be cleaner and since there are a lot of functions already provided by neovim, we wouldn't have to make our own logic there.

Furthering this idea, we wouldn't need a ```FolderEditor``` to have a neoviminstance at all, since we technically wouldn't even <i>need</i> to ever contact neovim through the folder interface (assuming we implement our own.)